services/TipiVariable.py

```diff
@@ -184,11 +186,11 @@ def store(self, caller_guid):
             
         f.close()        
         
-        
-        
 
     def handle(self, bytes):
```

Comment:
Note, we could speed up CHATTI and other raw extension use of TipiVariable by having this method loop until message is a QUIT command or something. That way you wouldn't have to send the small but expensive 'enter tipi variables' message before each tipi variable data message. You would just have the PI in TipiVariable mode. 

CHATTI might not care so much... except it might help shorten the polling time that is disturbing people. COMBATTI will probably see a large improvement from a change like that.

Reply:
Ok, I misunderstood about the desired location for .tipivars - cool.